### PR TITLE
Fix rpc_benchmark and add fcall mode

### PR DESCRIPTION
**Pull request by nb-ohad**
### Explain the changes
1. Fix rpc_banchmark (was broken since https://github.com/noobaa/noobaa-core/pull/6022 was merged)
2. Refactor start as an async/await function
3. Add a `--fcall` flag to test RPC over `fcalll` connection (process acts as both server and client)

### Issues: Fixed #xxx / Gap #xxx
1. 

### Testing Instructions:
1.

### CODE REVIEW

File: src/rpc/rpc_benchmark.js [hunk 1]
```diff
@@ -110,14 +117,15 @@ schema.compile();
 
 // create rpc
 const rpc = new RPC({
-    schema: schema,
+    client_factory: api.client_factory_from_schema(schema),
```
Comment (guymguym) [inline src/rpc/rpc_benchmark.js:120]: Hmm, I want to make sure `rpc` module is usable even without `api` module.
Do you think this can be done?

**Review by guymguym — APPROVED** [review]
See one comment, but overall this is good. Any results from running it?

File: src/rpc/rpc_benchmark.js (hunk 1 above)
Comment (nb-ohad) [reply src/rpc/rpc_benchmark.js:120]: Ok we have it now in a sepetate PR: https://github.com/noobaa/noobaa-core/pull/6178
We can wait to merge this one untill the other one is merged

**Review by guymguym — CHANGES_REQUESTED** [review]
Please rebase master

File: src/rpc/rpc_benchmark.js (hunk 1 above)
Comment (nb-ohad) [reply src/rpc/rpc_benchmark.js]: Rebased against `master` and removed dependency with the `api` module

**Review by guymguym — APPROVED** [review]
Looks great. Do you have the results of running this between the new fcall that clones messages and the previous which was using JSON.stringify/parse?

**Comment by nb-ohad** [comment]
No, as this change was merged to master after the fcall changes

**Pull request merged**